src/bpftrace.cpp

```diff
@@ -1477,73 +1497,6 @@ int BPFtrace::print_map_hist(const BpfMap &map, uint32_t top, uint32_t div)
   return 0;
 }
 
-int BPFtrace::print_map_stats(const BpfMap &map, uint32_t top, uint32_t div)
-{
-  uint64_t nvalues = map.is_per_cpu_type() ? ncpus_ : 1;
-  // stats() and avg() maps add an extra 8 bytes onto the end of their key for
-  // storing the bucket number.
-
-  auto maybe_old_key = find_empty_key(map);
-  if (!maybe_old_key.has_value()) {
-    return -2;
-  }
-  auto old_key(std::move(*maybe_old_key));
-  auto key(old_key);
-
-  std::map<std::vector<uint8_t>, std::vector<int64_t>> values_by_key;
-
-  const auto &map_key = resources.maps_info.at(map.name()).key;
-  while (bpf_get_next_key(map.fd(), old_key.data(), key.data()) == 0) {
-    auto key_prefix = std::vector<uint8_t>(map_key.size());
-    uint64_t bucket = read_data<uint64_t>(key.data() + map_key.size());
-
-    for (size_t i = 0; i < map_key.size(); i++)
-      key_prefix.at(i) = key.at(i);
-
-    auto value = std::vector<uint8_t>(map.value_size() * nvalues);
-    int err = bpf_lookup_elem(map.fd(), key.data(), value.data());
-    if (err == -ENOENT) {
-      // key was removed by the eBPF program during bpf_get_next_key() and
-      // bpf_lookup_elem(), let's skip this key
-      continue;
-    } else if (err) {
-      LOG(ERROR) << "failed to look up elem: " << err;
-      return -1;
-    }
-
-    if (values_by_key.find(key_prefix) == values_by_key.end()) {
-      // New key - create a list of buckets for it
-      values_by_key[key_prefix] = std::vector<int64_t>(2);
-    }
-    values_by_key[key_prefix].at(bucket) = reduce_value<int64_t>(value,
-                                                                 nvalues);
-
-    old_key = key;
-  }
-
-  // Sort based on sum of counts in all buckets
```

Comment:
I spent way too long trying to understand how this related to the code before I realised the comment was just blindly copied from `print_map_hist()` 🤦